src/System.Net.Http/tests/FunctionalTests/CancellationTest.cs

```diff
@@ -28,7 +28,7 @@ public CancellationTest(ITestOutputHelper output)
         [InlineData(false, true)]
         [InlineData(true, false)]
         [InlineData(true, true)]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "dotnet/corefx #19038")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "uap: dotnet/corefx #20010, netfx: dotnet/corefx #19038")]
```

Comment:
BTW: I believe ActiveIssue and SkipOnTargetFramework attributes can be combined. For example in this case, where .NetFramework's comment says it is a difference in behavior that we might not fix, we could leave the SkipOnTargetFramework for .NetFramework and add also `[ActiveIssue(20010, TargetFrameworkMonikers.Uap)]` (I have a PR open to unify Uap and UapAot where Uap will disable both) : https://github.com/dotnet/buildtools/pull/1506